<launch>
<!-- Perception: point cloud generation using depth_image_proc-->
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />
  <node pkg="nodelet" type="nodelet"
        name="nodelet1"
        args="load depth_image_proc/point_cloud_xyz nodelet_manager">
    <remap from="camera_info" to="/unity_ros/OurCar/Sensors/DepthCamera/camera_info"/>
    <remap from="image_rect" to="/unity_ros/OurCar/Sensors/DepthCamera/image_raw"/>
    <remap from="points" to="points_cloud"/>
    <param name="queue_size" value="10" />
  </node>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find perception_pkg)/cfg/point_cloud.rviz"/>

src/simulation/cfg/point_cloud.rviz
</launch>
